### PR TITLE
[ENG-353] remove dependency on other bundles when rendering datatables in Sourc…

### PR DESCRIPTION
**Pull request by scottshipman**
…e bundle

### CODE REVIEW

File: Assets/js/05.budget_widget.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    // danger or success
+    var forecastClass = 'success';
+    var forecastInt = parseInt(forecast);
+    if(forecastInt >=90){
+        forecastClass = 'danger';
+    }
+    return '<span class="label label-' + forecastClass + '">' + value + '</span>&nbsp;<span class="label label-' + forecastClass + '">' + forecast + '</span>';
+}
+
+mQuery(document).ready(function () {
```
Comment (heathdutton) [inline Assets/js/05.budget_widget.js]: Didn't we discover that we shouldn't be using document.ready()?

File: Assets/js/05.budget_widget.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+                    });
+                    mQuery('#budget-widget-overlay').hide();
+                    mQuery('#budgets-widget_wrapper .dt-buttons').css({float: "right", marginLeft: "10px"});
+                }
+            });
+        });
+    }
+};
+
+function renderStatusToggle (icon) {
```
Comment (heathdutton) [inline Assets/js/05.budget_widget.js]: Can we move these 3 functions inside the parent above, or just make them callbacks?

File: Assets/js/05.budget_widget.js [hunk 3]
```diff
@@ -0,0 +1,99 @@
+Mautic.loadCampaignBudgetsWidget = function () {
```
Comment (heathdutton) [inline Assets/js/05.budget_widget.js]: Can we namespace this under the plugin like all the other methods please?

File: Assets/js/05.budget_widget.js (hunk 1 above)
Comment (scottshipman) [reply Assets/js/05.budget_widget.js]: yes. didnt seem to hurt anything, but the whole things seems unnecessary now.

File: Assets/js/05.budget_widget.js (hunk 2 above)
Comment (scottshipman) [reply Assets/js/05.budget_widget.js]: It Shall be done

**Pull request merged**